templates/web/base/report/_item_expandable.html

```diff
@@ -0,0 +1,81 @@
+[%#
+  This list item contains a full photo set and description, which is hidden
+  until the `.expanded` class is added. It is useful for times when you want
+  to surface the full information about a report, without reqiring the user
+  to visit the dedicated report page.
+
+  Since the photo set includes `<a>` elements of its own, we drop the usual
+  "wrapper" `<a>` and the associated `.item-list--reports__item` class, to
+  avoid anchor nesting.
+%]
+
+<li class="item-list__item item-list__item--expandable [% item_extra_class %]"
+    data-report-id="[% problem.id | html %]"
+    data-lastupdate="[% problem.lastupdate %]"
+    id="report-[% problem.id | html %]">
+
+  [% IF problem.photo %]
+    <a href="[% c.cobrand.base_url_for_report( problem ) %][% problem.url %]" class="item-list__item--expandable__hide-when-expanded">
+        <img class="img" height="60" width="90" src="[% problem.photos.first.url_fp %]" alt="">
+    </a>
+  [% END %]
+
+  [% TRY %]
+    [% PROCESS 'report/_item_heading.html' %]
+  [% CATCH file %]
+    <h3>
+        <a href="[% c.cobrand.base_url_for_report( problem ) %][% problem.url %]">
+            [% problem.title | html %]
+        </a>
+    </h3>
+  [% END %]
+
+    <small>
```

Comment:
All the `small` is the same in `_item.html` and should be factored out to a new shared template - makes it much easier to then compare to the main one.